remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RulesImpl.java

```diff
@@ -130,6 +132,26 @@ public void deleteRuleById(String folderNodeId, String ruleSetId, String ruleId)
         ruleService.removeRule(folderNodeRef, rule);
     }
 
+    @Override
+    public RuleExecution executeRules(final String folderNodeId, final boolean eachSubFolderIncluded, final boolean eachInheritedRuleExecuted)
+    {
+        final NodeRef folderNodeRef = validator.validateFolderNode(folderNodeId, false);
+        final Map<String, Serializable> parameterValues = new HashMap<>();
+        parameterValues.put(ExecuteAllRulesActionExecuter.PARAM_RUN_ALL_RULES_ON_CHILDREN, eachSubFolderIncluded);
+        parameterValues.put(ExecuteAllRulesActionExecuter.PARAM_EXECUTE_INHERITED_RULES, eachInheritedRuleExecuted);
+        final ActionImpl action = new ActionImpl(null, GUID.generate(), ExecuteAllRulesActionExecuter.NAME);
+        action.setNodeRef(folderNodeRef);
+        action.setParameterValues(parameterValues);
+
+        ActionAccessRestriction.setActionContext(action, ActionAccessRestriction.V1_ACTION_CONTEXT);
+        actionService.executeAction(action, folderNodeRef, true, false);
```

Comment:
Could/should we check if at the moment any actions on given folder(s) are being executed (maybe this is somehow already happening in `actionService.executeAction` - don't know)?
In some cases action executions take quite some time and running them again could not be what we want.